.circleci/docker-compose-prod.yml

```diff
@@ -1,39 +1,56 @@
-version: '3'
+version: '3.1'
+
 services:
   couch:
     image: "couchbase:5.1.0"
-    ports:
-      - "8091-8094:8091-8094"
-      - "11210-11211:11210-11211"
-    container_name: "couch-dakar"
     volumes:
       - couch:/opt/couchbase/var
     networks:
       - dakar-network
-  app:
+      - traefik_traefik
+    deploy:
+      labels:
+        - "traefik.enable=true"
+        - "traefik.http.routers.dakar-couch.rule=Host(`dakar-couch.project.lambla.eu`)"
+        - "traefik.http.routers.dakar-couch.entrypoints=web"
+        - "traefik.http.services.dakar-couch.loadbalancer.server.port=8091"
+
+  api:
     image: "dakarinternational/dakar:latest"
-    ports:
-      - "8080:8080"
-    container_name: 'app-dakar'
     depends_on:
       - "couch"
     networks:
       - dakar-network
     environment:
       PROFILE: prod
-  app-ng:
+
+  front:
     image: "dakarinternational/dakar-ng:latest"
-    ports:
-      - "80:80"
-    container_name: 'app-dakar-ng'
     depends_on:
-      - "app"
+      - "dakar-api"
```

Comment:
if think that you just mean "api"?
But why the front is depending on the backend?

Reply:
dependenci is non mandatory, you're right. it's fixed.